tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py

```
@@ -31,353 +32,285 @@ class TestFailoverAndRelocateWithDiscoveredApps:
     """
 
     @pytest.mark.parametrize(
-        argnames=[
-            "primary_cluster_down",
-            "pvc_interface",
-            "kubeobject",
-            "recipe",
-            "iterations",
-        ],
+        argnames=["primary_cluster_down", "pvc_interface"],
```

Ruff (0.15.15)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)